### PR TITLE
Re adds short labcoats and wintercoats to departmental secure lockers.

### PR DESCRIPTION
**Pull request by BananaCreamPie**
Just readds the short labcoats IN ADDITION TO neinhaus' new longer sprited versions. Also added wintercoats to secure department lockers.

### CODE REVIEW

**Comment by Chocobro** [comment]
:+1: looks good

**Comment by WJohn** [comment]
MAXIMUM

VARIETY

File: tgstation.dme [hunk 1]
```diff
@@ -6,6 +6,52 @@
...
+#define FILE_DIR "sound/magic"
+#define FILE_DIR "sound/mecha"
+#define FILE_DIR "sound/misc"
+#define FILE_DIR "sound/piano"
+#define FILE_DIR "sound/spookoween"
+#define FILE_DIR "sound/violin"
+#define FILE_DIR "sound/voice"
+#define FILE_DIR "sound/voice/complionator"
+#define FILE_DIR "sound/vox_fem"
+#define FILE_DIR "sound/weapons"
```
Comment (phil235) [inline tgstation.dme]: I don't think you meant to change that...

**Comment by Razharas** [comment]
@BananaCreamPie fix merge conflicts and remive the changes to the dme

**Comment by Ikarrus** [comment]
Lockers and wardrobes have changed since so this PR is outdated anyways.

**Comment by Razharas** [comment]
No reaction for over a week + no activity for a few weeks before that - ask if you want it reopened

**Pull request closed without merging**